blocks/init/src/blocks/components/hamburger/hamburger.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Template for the Button Component.
+ *
+ * @package Eightshift_Boilerplate\Components.
+ */
+
+namespace Eightshift_Libs\Blocks;
+
+$block_class   = $attributes['blockClass'] ?? 'hamburger';
+$trigger       = $attributes['trigger'] ?? '';
+$overlay_class = $attributes['overlayClass'] ?? '';
+?>
+
+<button
+  data-trigger="<?php echo esc_attr( $trigger ); ?>"
+  data-overlay="<?php echo esc_attr( $overlay_class ); ?>"
+  class="<?php echo esc_attr( $block_class ); ?> js-<?php echo esc_attr( $block_class ); ?>"
+>
+  <div class="<?php echo esc_attr( $block_class ); ?>__wrapper">
```

Comment:
By the HTML spec `divs` or any block elements should not be inside the `button`. You should use `spans` or some other inline elements (but not anchors) that can have `display: block` if its needed.

https://stackoverflow.com/questions/12982269/is-it-semantically-incorrect-to-put-a-div-or-span-inside-of-a-button/48383389

Reply:
Huh, didn't know about this, fixed.